packages/wrangler/src/__tests__/helpers/mock-cfetch.ts

```diff
@@ -206,6 +209,48 @@ export function setMockFetchKVGetValue(
 	kvGetMocks.set(`${accountId}/${namespaceId}/${key}`, value);
 }
 
+/**
+ * @mocked typeof fetchR2Objects
+ */
+export async function mockFetchR2Objects(
+	resource: string,
+	bodyInit: {
+		body: BodyInit | Readable;
+		headers: HeadersInit | undefined;
+		method: "PUT" | "GET";
+	}
+): Promise<Response> {
+	if (bodyInit.body instanceof Readable) {
+		bodyInit.body.destroy();
+		bodyInit.body.removeAllListeners();
+	}
+
+	if (r2GetMocks.has(resource)) {
+		const value = r2GetMocks.get(resource);
+
+		return new Response(value);
+	}
+	throw new Error(`no expected mock found for \`r2 object get\` - ${resource}`);
+}
+
+/**
+ * Mock setter for usage within test blocks, companion helper to `mockFetchR2Objects`
+ */
+export function setMockFetchR2Objects(
+	{
+		accountId = "r2-object-test-account",
+		bucketName = "r2-test-bucket",
+		objectName = "r2-test-object",
+	},
+	value?: string
```

Comment:
How about you move this parameter inside the first parameter e.g.:

```ts
export function setMockFetchR2Objects(
	{
		accountId = "r2-object-test-account",
		bucketName = "r2-test-bucket",
		objectName = "r2-test-object",
    mockResponse?: string,
	},
) { ... }
```

Reply:
Sure, I don't have strong opinion on the param structure.